src/WebJobs.Script/Description/Node/Script/functionTemplate.js

```diff
@@ -40,4 +42,36 @@ return function (context, callback) {{
     delete context.inputs;
 
     f.apply(null, inputs);
-}};
\ No newline at end of file
+}};
+
+function getEntryPoint(f, context) {{
+    if (util.isObject(f)) {{
```

Comment:
I created https://github.com/Azure/azure-webjobs-sdk-script/issues/481 to track that